.travis.yml

```diff
@@ -18,3 +18,11 @@ jobs:
         - yarn lint
         - yarn build
         - yarn test
+      deploy:
+        provider: pages
+        local_dir: docs/build
+        skip_cleanup: true
+        github_token: $GITHUB_TOKEN  # Set in the settings page of your repository, as a secure variable
+        keep_history: true
+        on:
+          branch: PPCDEV-12705-generator
```

Comment:
How about to create more generic branch for deployment? `deploy/release`?

Reply:
just to test, deploy will be only from master